wallet/how-to/connect/detect-metamask.md

```diff
@@ -5,6 +5,16 @@ sidebar_position: 1
 
 # Detect MetaMask
```

Comment:
Should we still document this if we no longer recommend users connect using this legacy method?

Reply:
The tutorials still use this method and link to this page. Those will updated in a future PR. We also still want to document this method for a beginner use case – may move this to a tutorial in the future.